dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassReferenceGraph.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import com.google.common.graph.Graph;
+import com.google.common.graph.GraphBuilder;
+import com.google.common.graph.MutableGraph;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayDeque;
+import java.util.Collection;
+import java.util.Queue;
+import java.util.Set;
+import org.apache.bcel.classfile.JavaClass;
+
+/**
+ * Directed graph of class references. Given classes in a set of entry points, it provides {@link
+ * #isReachable(String)} for a class to check the reachability. The graph's node and edges are
```

Comment:
This might be a good place to define reachability; either here or in README.md or both

Reply:
I would let README.md define the term and adding `@see` link from this to the doc.